webapi/src/main/scala/org/knora/webapi/slice/admin/domain/model/GroupIri.scala

```diff
@@ -13,12 +13,14 @@ import dsp.valueobjects.UuidUtil
 import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.BuiltInGroups
 import org.knora.webapi.slice.admin.domain.model.KnoraProject.ProjectIri
 import org.knora.webapi.slice.admin.domain.model.KnoraProject.Shortcode
+import org.knora.webapi.slice.common.StringValueCompanion
+import org.knora.webapi.slice.common.Value.StringValue
 
 final case class KnoraUserGroup(id: GroupIri, belongsToProject: ProjectIri)
 
-final case class GroupIri private (value: String) extends AnyVal
+final case class GroupIri private (override val value: String) extends AnyVal with StringValue
 
-object GroupIri {
+object GroupIri extends StringValueCompanion[GroupIri] {
 
   implicit val tapirCodec: Codec[String, GroupIri, CodecFormat.TextPlain] =
```

Comment:
**suggestion (if-minor)**: This `Codec` also exists in our common codecs

Could we please check if this can be easily removed?

Reply:
It seems to be missed here `constructor GroupIri in class GroupIri cannot be accessed in object AdminPathVariables from object AdminPathVariables in package api
[error]     path[GroupIri]` so I keep it for now. At the end of Tapir migration, there should be general cleanup made, so all the path vars and codecs could be unified and moved to the same location across packages.